Arrangement_on_surface_2/include/CGAL/Arr_geometry_traits/Conic_x_monotone_arc_2.h

```diff
@@ -922,22 +838,22 @@ class _Conic_x_monotone_arc_2 : private Conic_arc_
    * \param oi The output iterator.
    * \return The past-the-end iterator.
    */
-  template<class OutputIterator>
-  OutputIterator intersect (const Self& arc,
-                            Intersection_map& inter_map,
-                            OutputIterator oi) const
+  template <typename OutputIterator>
+  OutputIterator intersect(const Self& arc,
+                           Intersection_map& inter_map,
+                           OutputIterator oi) const
   {
-    if (_has_same_supporting_conic (arc))
-    {
+    typedef unsigned int                              Multiplicity;
+    typedef boost::variant<Intersection_point, Self>  Intersection_result;
```

Comment:
There is a missing `#include <boost/variant.hpp>` in this file `Arrangement_on_surface_2/include/CGAL/Arr_geometry_traits/Conic_x_monotone_arc_2.h`. It was detected by [our automatic tests on Travis CI](https://travis-ci.com/github/CGAL/cgal/jobs/314578509#L7177).